pyvptree/include/VPTree.hpp

```diff
@@ -319,14 +192,14 @@ template <typename T, typename distance_type, distance_type (*distance)(const T
             searchKNN(_rootPartition, query, k, knnQueue);
 
             // we must always return k elements for each search unless there is no k elements
-            assert(static_cast<unsigned int>(knnQueue.size()) == std::min<unsigned int>(_examples.size(), k));
+            assert(static_cast<size_t>(knnQueue.size()) == std::min<size_t>(_examples.size(), k));
```

Comment:
the cast is not necessary anymore I think

Reply:
Removed on 4570bbfe2374d259dd5f2e8385aa1fe8ac3da9b5
Thanks.